CMakeLists.txt

```diff
@@ -22,7 +22,6 @@ include(EthCcache)
 # Let's find our dependencies
 include(EthDependencies)
 include(jsoncpp)
-include_directories(SYSTEM ${JSONCPP_INCLUDE_DIR})
```

Comment:
This isn't necessary, since we already use an imported target for jsoncpp.

Reply:
Ah, ok, it seems when not declaring this a system include directory, then macos still uses the wrong header - I guess setting the interface system include directory on the jsoncpp target will solve that as well, though.

Reply:
Ok, this seems to be a macos issue that's not easily solved. Adding it back.

Reply:
I think the way we deal with jsoncpp in CI may actually be problematic. But since I think we should change the way we deal with jsoncpp in general, I'll just keep this workaround for now.